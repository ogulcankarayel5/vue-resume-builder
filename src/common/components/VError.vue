<template>
  <p class="text-red-700 font-bold">{{ error }}</p>
</template>

<script lang="ts">
import Vue from "vue";

export default Vue.extend({
  name: "VError",
  props: {
    error: {
      type: String,
      required: true,
    },
  },
});
</script>
