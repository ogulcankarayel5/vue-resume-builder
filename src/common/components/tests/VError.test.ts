import VError from "@/common/components/VError.vue";
import { shallowMount } from "@vue/test-utils";
describe("VError", () => {
  it("snapshot", () => {
    const wrapper = shallowMount(VError);
    expect(wrapper.element).toMatchSnapshot();
  });
  it("should render the right message", () => {
    const wrapper = shallowMount(VError, {
      propsData: {
        error: "test",
      },
    });
    expect(wrapper.find("p").text()).toBe("test");
  });
});
