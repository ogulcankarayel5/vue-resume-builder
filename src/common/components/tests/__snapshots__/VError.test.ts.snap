// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VError snapshot 1`] = `
<p
  class="text-red-700 font-bold"
>
  
</p>
`;
